### DISCUSSION TITLE
Label not showing a newly connected reach.

### DISCUSSION THREAD

**Discussion by PeterZamudio** [QGEP/QGEP#822] [Q&A]
I have a problem that I have 2 reaches going in and 1 out. But the manhole label is only showing:
D=
S=
E=
A=
![image](https://github.com/QGEP/QGEP/assets/32352291/20f83034-777b-46f0-a09b-6ae880e52e33)
It should be showing two E=
I have checked all the reach and manhole connections to the reach points and nodes. They all seem correct. There is 3 reachpoints. I also disconnected and reconnected the networkelements and recalculated the topologies. There are 2 pipes going in and only one is showing the lable E=.
Does anyone know a trick to kick the labels into refreshing themselves.

We are using
QGIS 3.28.3 tried it with 3.30.3 as well and the newest QGEP Plugin
We also recently updated the Datamodel to 1.6.0
Label Exports are working fine too.

**Comment by urskaufmann** [comment]
I think its because these reaches/channels are not primary in function_hierarchic.
In FUNCTION: qgep_od.update_wastewater_structure_label is a condition for the labeling of inputs:
      WHERE (_all OR NE.fk_wastewater_structure = _obj_id) and CH_to.function_hierarchic in (5062,5064,5066,5068,5069,5070,5071,5072,5074)  ----label only reaches with function_hierarchic=pwwf.*
If you delete the second part of this WHERE clause, you will have a label for every reach, that is an input for the nodes of the manhole.

**Reply by PeterZamudio** [reply]
Hello Urs
Thank you so much. You are absolutely right there. It's working again as it should. I remember it now from 2 years ago. It must have come back in with the last PUM updates to 1.6.0

It will be good to have this issue here to remind and help people with the same problem.
